Arduino/TWatch 2.0/TWatch 2.0.ino

```diff
@@ -144,7 +146,14 @@ void setup() {
   ttgo->begin();
   ttgo->tft->setTextFont(1);
   ttgo->tft->fillScreen(TFT_BLACK);
-  ttgo->tft->setTextColor(TFT_WHITE, TFT_BLACK);  // Note: the new fonts do not draw the background colour
+  ttgo->tft->setTextColor(TFT_WHITE);  // Note: the new fonts do not draw the background colour
+  
+  // Turn on the backlight
+  ttgo->openBL();
+  tft = ttgo->tft;
+  u8f.begin(*tft);
+  u8f.setForegroundColor(TFT_WHITE);
+
```

Comment:
Please add some comments about your changes.
It doesn't seem to relevant with the issue description

Reply:
done